spec/colorls_spec.rb

```diff
@@ -1,7 +1,85 @@
 require 'spec_helper'
 
 RSpec.describe ColorLS do
+  FIXTURES = 'spec/fixtures'.freeze
```

Comment:
Add a global `subject { ColorLS::Flags.new(flags).process }` and define individual context-flags with `let`.